sklearn/_build_utils/openmp_helpers.py

```diff
@@ -0,0 +1,137 @@
+"""Helpers for OpenMP support during the build."""
+
+# This code is adapted for a large part from the astropy openmp helpers, which
+# can be found at: https://github.com/astropy/astropy-helpers/blob/master/astropy_helpers/openmp_helpers.py  # noqa
+
+
+import os
+import sys
+import glob
+import tempfile
+import subprocess
+
+from numpy.distutils.ccompiler import new_compiler
+from distutils.sysconfig import customize_compiler
+from distutils.errors import CompileError, LinkError
+
+
+CCODE = """
+        #include <omp.h>
+        #include <stdio.h>
+        int main(void) {
+        #pragma omp parallel
+        printf("nthreads=%d\\n", omp_get_num_threads());
+        return 0;
+        }
+        """
+
+
+def get_openmp_flag(compiler):
+    if os.getenv('SKLEARN_NO_OPENMP') == 'TRUE':
```

Comment:
I think, for binary env variables, their existence should interpreted as True, i.e. `if os.getenv('SKLEARN_NO_OPENMP')`. That removes the need to  think about how to define boolean True (or `"True"` string) in environment variable in BASH and Windows CMD: e.g. whether it's `true`, `"TRUE"`, `True` or maybe `'True'`...